### DISCUSSION TITLE
Iterative learning

### DISCUSSION THREAD

**Discussion by gentine** [stanfordmlgroup/ngboost#256] [General]
Would there be a simple way in NGBoost to do some iterative training of a previously trained model (ideally with a different learning rate and on a different dataset)?
Thanks a lot

**Comment by alejandroschuler** [comment]
I don't think there's a simple way to do this at the moment. To be clear, you're talking about adding base estimators to an already fit model, right? Something like:
```
model = NGBoostRegressor(n_estimators=100, learning_rate=0.1)
model.fit(X1, Y1)
model.add_estimators(
    X2, Y2, 
    params=dict(n_estimators=50, learning_rate = 0.01)
)
```
I brought this up briefly in another recent discussion (https://github.com/stanfordmlgroup/ngboost/discussions/253). If you're interested in making a PR, I don't think it would be terribly challenging and it would be much appreciated!

**Comment by gentine** [comment]
Hi Alejandro,
Yes this is exactly what I meant. 
I will try to to make a PR.
